mozilla/activity-stream
# Bug 1585147 - Migrate AS Router telemetry to Structured Ingestion (GCP)

Gear switch from AWS to GCP for ASRouter telemetry.

Steps to test:
* Enable PingCentre logging by flipping `browser.ping-centre.log`
* Enable telemetry for Activity Stream by flipping `browser.newtabpage.activity-stream.telemetry`
* Enable Structured Ingestion by flipping `browser.newtabpage.activity-stream.telemetry.structuredIngestion` 

Then you should be able to see ping logging in the browser console.
